projects/microphysics/scripts/score_training.py

```diff
@@ -48,36 +51,55 @@ def main(config: TrainConfig, seed: int = 0, model_url: str = None):
         config.wandb.init(config=d)
 
     if model_url is None:
-        model = load_final_model_or_checkpoint(config.out_url)
+        model, model_url = load_final_model_or_checkpoint(config.out_url)
     else:
         logger.info(f"Loading user specified model from {model_url}")
         model = tf.keras.models.load_model(model_url)
 
+    StepMetadata(
+        job_type="train_score",
+        url=config.out_url,
+        dependencies=dict(
+            train_data=config.train_url, test_data=config.test_url, model=model_url
+        ),
+        args=sys.argv[1:],
+    ).print_json()
+
     train_ds = config.open_dataset(
         config.train_url, config.nfiles, config.model_variables
     )
     test_ds = config.open_dataset(
         config.train_url, config.nfiles, config.model_variables
     )
 
-    train_set = next(iter(train_ds.unbatch().shuffle(100_000).batch(50_000)))
+    train_set = next(iter(train_ds.unbatch().shuffle(160_000).batch(80_000)))
```

Comment:
likewise, please isolate this into a separate commit or remove from the PR for now.